examples/c/test_tracker.c

```diff
@@ -44,7 +44,11 @@
 int main(int arg, char** args) {
     int i;
     int count = psmove_count_connected();
+#ifdef _MSC_VER
+    PSMove* controllers[8];
```

Comment:
Thank you. This is very helpful. I tried malloc and couldn't get it to work so I just made a quick hack  for testing purposes. Again, I forgot I was working on a branch with an existing pull request. Sorry.